### DISCUSSION TITLE
Error "invalid repository" when pushing

### DISCUSSION THREAD

**Discussion by jjcaballero** [oras-project/oras#930] [Q&A] [answered]
When I do:

`$ oras repo ls <registry>/folder/subfolder`

I can see the content under `folder/subfolder`

But if I try to push something on that folder, I get invalid repository:

```
$ oras push <registry>/folder/subfolder/newFile:v1 localFile.txt
Error: invalid reference: invalid repository

$ cat localFille.txt
this is a dummy file
```

What can cause this? I have tried different scenarios: when subfolder exists, when subfolder does not exist, giving artifact type, etc.

I understand that an user with write permissions should have permissions to create the subfolder and new file, right?

Any hint will be welcome. Thank you!

**Comment by qweeah** [comment, marked as answer]
repository only allow lower-case letter. new**F**ile is invalid.

**Reply by jjcaballero** [reply]
that worked! thanks @qweeah
